engine.py

```diff
@@ -22,6 +22,18 @@ class UnityEditorEngine(Engine):
     and that all logging calls to Toolkit logging methods will be forwarded there.
     """
 
+    @property
```

Comment:
This can be removed now. it was a temporary stopgap measure, but we're supposed to have solved it.